.github/workflows/shellcheck.yml

```diff
@@ -11,8 +13,6 @@ jobs:
     - name: Run ShellCheck
       uses: ludeeus/action-shellcheck@master
       env:
-        SHELLCHECK_OPTS: -e SC1091 -e SC2034 -e SC2154 -e SC3040 -e SC2148
+        SHELLCHECK_OPTS: -x
       with:
         scandir: '.'
```

Comment:
Check together etc. not needed any more.

@bretton As we had missing newline at eof quite frequently, this site gives useful advise for many editors (and why it's important and github marks it in a quite ugly way): https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
think I got confused here and previous request was "no newline" and I added it to style guide on repo wiki :)

Reply:
I didn’t request "no newline", but I tried to point out that there was no newline :) The article is worth reading anyway, as it points out why POSIX requires it - the site has also more interesting information, e.g. how "grep" got its name.